kalliope/core/RestAPI/FlaskAPI.py

```diff
@@ -39,7 +39,7 @@ def _get_synapse_by_name(self, synapse_name):
         """
         all_synapse = self.brain.brain_yaml
         for el in all_synapse:
-            print el
+            print(el)
```

Comment:
In python 2 it's not a syntax error. My IDE doesn't alert me about any error anyway. Did you configure you IDE to use python 2.7 interpreter?